### DISCUSSION TITLE
How can I change the font and font size in the terminal window?

### DISCUSSION THREAD

**Discussion by ericfitz** [hassio-addons/addon-ssh#818] [General]
The font in the terminal in the console is too small.  How can I change this?  I wasn't able to find a configuration setting.

Thank you,
Eric

**Comment by patrikgafvert** [comment]
I know that the file https://github.com/hassio-addons/app-ssh/blob/8f845118f407616b989777436d42f2e3ed9cc32e/ssh/rootfs/etc/s6-overlay/s6-rc.d/ttyd/run#L42

Has to take some parameters to ttyd. 
ttyd -t fontSize=16 -t fontFamily="Consolas, Monaco, monospace"
